src/cable/connection.cr

```diff
@@ -52,6 +52,14 @@ module Cable
       @connection_rejected = true
     end
 
+    def connection_accepted? : Bool
+      !connection_rejected?
+    end
+
+    def open? : Bool
+      !closed?
+    end
```

Comment:
helper methods to avoid using the negative `unless closed?` or `if !closed?`

Reply:
These methods aren't needed anymore if the one-liner check in the suggestion below is used instead.